_episodes/10-large-data.md

```diff
@@ -0,0 +1,343 @@
+---
+title: "Using dask to handle larger data"
+teaching: 30
+exercises: 30
+questions:
+- "How do I work with multiple CMIP files that won't fit in memory?"
+objectives:
+- "Import the dask library and start a client with parallel workers."
+- "Inspect netCDF chunking."
+- "Calculate and plot an ocean temperature climatology."
+keypoints:
+- "Libraries such as dask and xarray can make loading, processing and visualising netCDF data much easier."
+- "Dask can massively speed up processing through parallelism but care may be needed particularly with data chunking."
+---
+
+When working with climate data, it is common to find we want to analyse more data than can fit in memory at one time, or that calculations take a long time. We may be able to parallelise our operations with the `dask` library, which is built on `xarray` and enables calculations to make use of multiple "cores" in our computer.
+
+> ## Additional libraries
```

Comment:
At the moment we install all the required libraries up front in the "Software installation for these lessons" section of the first chapter, so for consistency we should probably install `siphon` and `dask` there. (We'll also need to add them to the software installation instructions in the Setup menu.)

Reply:
Yes, I am happy to move the requirements into there, just wanted to wait until this lesson was reviewed in case we don't want to use either or both (though realistically I think we *do* want dask at least!).